quickref/qsharp-quick-reference.tex

```diff
@@ -782,7 +854,60 @@
     $ R_1(\theta)\ket{1} = e^{i\theta}\ket{1} $  &  \\
 
   \end{keysrefGates}
+  
+  \begin{keysrefMultiGates}{Two-qubit gates}
+  
+    Gate &
+    Matrix Representation & 
+    Ket-Bra Representation &
+    Applying to $\ket{\psi} = \alpha\ket{00} + \beta\ket{01} + \gamma\ket{10} + \delta\ket{11}$ &
+    Applying to basis states & \\
+    
+    CNOT &
+    \usebox\cnotgate & 
+    \usebox\braketcx &
+    $ \text{CNOT}\ket{\psi} = \alpha\ket{00} + \beta\ket{01} + \delta\ket{10} + \gamma\ket{11}$ &
+    $ \text{CNOT}\ket{00} = \ket{00} \newline
+      \text{CNOT}\ket{01} = \ket{01} $ &
+    $ \text{CNOT}\ket{10} = \ket{11} \newline
+      \text{CNOT}\ket{11} = \ket{10} $ \\
+ 
+    SWAP &
+    \usebox\swapgate & 
+    $ \ket{00}\bra{00} + \ket{01}\bra{10} + \ket{10}\bra{01} + \ket{11}\bra{11} $ &
+    $ \text{SWAP}\ket{\psi} = \alpha\ket{00} + \gamma\ket{01} + \beta\ket{10} + \delta\ket{11}$ &
+    $ \text{SWAP}\ket{00} = \ket{00} \newline
+      \text{SWAP}\ket{01} = \ket{10} $ &
+    $\text{SWAP}\ket{10} = \ket{01} \newline
+      \text{SWAP}\ket{11} = \ket{11} $ \\
+      
+    Controlled U &
+    \usebox\controlledugate  & 
+    \usebox\braketcu & 
+    $ \text{C(U)} \ket{\psi} =  \alpha\ket{00} + \gamma\ket{01} + (\gamma  a_{0,0} + \delta  a_{0,1}) \ket{10} +  (\gamma  a_{1,0} + \delta  a_{1,1}) \ket{11}  $ & 
+    $ \text{C(U)} \ket{00} = \ket{00} \newline
+      \text{C(U)} \ket{01} = \ket{01} $ & 
+    $\text{C(U)} \ket{10} = \ket{1} \otimes (\alpha \ket{0} + \gamma \ket{1}) \newline \text{C(U)} \ket{11} = \ket{1} \otimes (\beta \ket{0} + \delta \ket{1})  $ \\
```

Comment:
I think the result of applying C(U) to the basis states should have a_00, a_01 etc in the expression, not alpha and beta - those were coefficients of the psi state, not of the U matrix